packages/p2p-media-loader-demo/src/declaration.d.ts

```diff
@@ -0,0 +1,14 @@
+import { HlsJsP2PEngine } from "p2p-media-loader-hlsjs";
+import Hls from "hls.js";
+
+const HlsWithP2P = HlsJsP2PEngine.injectMixin(Hls);
+declare global {
+  interface Window {
+    shaka?: unknown;
+    Hls?: typeof HlsWithP2P;
+    LevelSelector: unknown;
+    DashShakaPlayback: unknown;
+  }
+}
+
+export {};
```

Comment:
it seems like I declared right type for Hls, but I get TS errors across all files where window.Hls is used.
I can't find a good solution to those errors: " error TS2339: Property 'Hls' does not exist on type 'Window & typeof globalThis'."

```  
 // @ts-ignore
 window.Hls = HlsJsP2PEngine.injectMixin(Hls);
...
 // @ts-ignore
window.Hls = undefined;
 ```